### ISSUE TITLE
Cannot install Jupyter C# kernel with the provided dockerfile - error Microsoft.PowerShell.Native not found 

### ISSUE THREAD

**Issue by mdfarragher** [dotnet/interactive#109]
#### Describe the bug
I'm trying to run my own jupyter notebook using MyBinder. I have copied your dockerfile and nuget.config over into my repo and am trying to start the container with mybinder. The installation progresses up until the point where the jupyter kernel gets installed, and then I get the following error message:

Error:
  An assembly specified in the application dependencies manifest (Microsoft.DotNet.Interactive.App.deps.json) was not found:
    package: 'Microsoft.PowerShell.Native', version: '7.0.0-rc.1'
    path: 'runtimes/linux-x64/native/libmi.so'

My mybinder url is: https://mybinder.org/v2/gh/mdfarragher/GLC0220/master?filepath=%2Fblob%2Fmaster%2FRegression%2FTaxiFarePrediction%2FTaxiFarePrediction.ipynb

And my repo is here: https://github.com/mdfarragher/GLC0220

#### Did this error occur while using `dotnet interactive`?
- [x] .NET Jupyter Notebook 

#### Screenshots

![Annotation 2020-02-03 111409](https://user-images.githubusercontent.com/18468174/73644963-a2d25500-4676-11ea-96ac-1ea06b35f3cc.png)

#### Please complete the following:
 - OS
    - [X] Windows 10
    - [ ] macOS
    - [ ] Linux (Please specify distro)
    - [ ] iOS
    - [ ] Android
 - Browser 
    - [ ] Chrome
    - [X] Edge (the new one)
    - [ ] Safari

**Comment by jonsequitur** [comment]
@brettfo

**Comment by brettfo** [comment]
The issue is that two identical`*` packages were published to MyGet and MyGet handles this by overwriting the package.  The 'identical' package was really a symbols package `*.symbols.nupkg` that was specially created for internal symbol archiving purposes, but due to a limitation in the tool, two binaries had to be stripped out; `libmi.so` and `libpsl-native.so`.

I've created PR #113 that publishes to MyGet using a build artifact that doesn't have the symbols packages, so we won't run into the overwrite problem.  Once that PR has merged, an internal signed build will be automatically started which will push new (and correct) packages to MyGet.  ETA less than 1 hour from this comment.

**Comment by brettfo** [comment]
**tl;dr** - The MyGet packages have been fixed.  Try to re-generate your MyBinder image and as long as the version of dotnet-interactive that's installed is 1.0.110308 or greater, it should work.

The latest package is up on MyGet and I was able to verify it's behavior by running the following under Ubuntu:

``` bash
dotnet tool install -g Microsoft.dotnet-interactive --add-source "https://dotnet.myget.org/F/dotnet-try/api/v3/index.json"
```

`Tool 'microsoft.dotnet-interactive' (version '1.0.110308') was successfully installed.`

I was then able to force execution of a simple PowerShell command with the following:

`dotnet interactive kernel-server`

Paste in this:

`{"token":"abc","commandType":"SubmitCode","command":{"code":"1+1","submissionType":0,"targetKernelName":"powershell"}}`

And in the output spew, saw that the value `2` was computed.  I could also re-try the console submission but with `"targetKernelName":null` to default to C# and it also did the appropriate thing.

**Comment by brettfo** [comment]
Note, until mybinder clears their docker cache, this will still fail to build with the default Dockerfile.  You'll be able to tell because the log under the line `RUN dotnet tool install ...` you'll see `---> Using cache`.  You should be able to work around this by adding `--version 1.0.110308` to the install line.

**Comment by brettfo** [comment]
Looks like the mybinder cache has been updated, so it should work now.

**Comment by mdfarragher** [comment]
I can confirm it's fixed. I just added the version flag to my dockerfile and everything works perfectly now. Thanks for the fix @brettfo!
